content/en/armory-enterprise/feature-status/armory-enterprise-matrix/_index.md

```diff
@@ -172,12 +172,13 @@ Terraform Integration gives you the ability to use Terraform within your pipelin
 
 The following table lists the supported Terraform versions:
 
-| Terraform Versions | Armory Enterprise Version                 | Note |
-| ------------------ | ---------------------- | ---- |
-| 0.11.10 - 0.11.14  | All supported versions |      |
-| 0.12.0 - 0.12.24   | All supported versions |      |
-| 0.13.4 - 0.13.5   | 2.24.0 or later |      |
-| 0.14.0 - 0.14.2   | 2.24.0 or later |      |
+| Terraform Versions | Armory Enterprise Version | Note |
+| ------------------ | ------------------------- | ---- |
+| 0.11.10 - 0.11.15  | 2.27.0 and earlier        |      |
+| 0.12.0 - 0.12.31   | All supported versions    |      |
+| 0.13.4 - 0.13.5    | 2.24.0 - 2.28.0           |      |
```

Comment:
Sorta minor, but probably should match on language.  E.g. "All supported versions" for all the columns vs. mentioning 2.24 and such?  

Another option:

| CDSH Version | Terraform Supported Versions | Notes |
| ----- | ----- | ---- | 
| 2.26 |  0.11, 0.12, 0.13, 0.14, 0.15, 1.0 |
| 2.27 |  0.11, 0.12, 0.13, 0.14, 0.15, 1.0, 1.1, 1.2 |
| 2.28 |  0.12, 0.13, 0.14, 0.15,  1.0, 1.1, 1.2 | 0.12 was removed initially and added back in 2.28.1 |
<td colspan=2>Note that not all patch versions are included, based upon request/time of release of available versions